daemon/mgr/network.go

```diff
@@ -528,11 +528,11 @@ func networkOptions(create apitypes.NetworkCreateConfig) ([]libnetwork.NetworkOp
 	}
 
 	// parse options
-	if v, ok := networkCreate.Options["persist"]; ok && v == "true" {
+	if v, ok := networkCreate.Options["persist"]; ok && v == trueString {
 		nwOptions = append(nwOptions, libnetwork.NetworkOptionPersist(true))
 		delete(networkCreate.Options, "persist")
 	}
-	if v, ok := networkCreate.Options["dynamic"]; ok && v == "true" {
```

Comment:
?

Reply:
In the "mgr" package, "true" appears three times, so I change it to a constant.

Reply:
but in my opnion this change didn't make code more readable.

Reply:
Yeah, I'll take your advise.